lib/src/test_signing_backend.rs

```diff
@@ -42,17 +60,9 @@ impl SigningBackend for TestSigningBackend {
 
         let sig = self.sign(data, key.as_deref())?;
         if sig == signature {
-            Ok(Verification {
-                status: SigStatus::Good,
-                key,
-                display: None,
-            })
+            Ok(Verification::new(SigStatus::Good, key, Some("test".into())))
```

Comment:
nit: since `backend` field is public, we don't have to use `::new()`. It's nice if field names are explicitly listed.

Reply:
I suppose that comment applies to all occurrences of `Verification::new()` then?
(We have a bunch more in `lib/src/gpg_signing.rs` and `lib/src/ssh_signing.rs`, for example.)

Reply:
Yes, but I wouldn't rewrite the existing callers within this commit. They're unrelated change.

Reply:
I will take some time and play with both approaches (`backend` private vs. public). I have this itchy feeling that leaving it private and adjusting the failing tests might be less work.

Reply:
or just remove `backend` for now? I don't think it's super important to know whether the signature was verified by `gpg` or `ssh-keygen`.

Reply:
It turned out to be very straightforward to update the tests with `backend` being private.
[insta's inline snapshots](https://insta.rs/docs/snapshot-types/#inline-snapshots) are *awesome*!

Reply:
We do introduce a little duplication, because we inline `good_signature()`, but I think that's just unavoidable with inline snapshots. We might be able to use [unnamed snapshots](https://insta.rs/docs/snapshot-types/#file-snapshots) as an alternative, but I liked that one can see the expected value when looking at the code.